lib/SILOptimizer/Mandatory/TFConstExpr.h

```diff
@@ -24,14 +24,16 @@
 #ifndef SWIFT_SILOPTIMIZER_TF_CONSTEXPR_H
 #define SWIFT_SILOPTIMIZER_TF_CONSTEXPR_H
 
-#include "swift/SIL/SILConstants.h"
 #include "llvm/Support/Allocator.h"
+#include "swift/Basic/LLVM.h"
```

Comment:
What does this include do?

Reply:
Provides forward declarations for ArrayRef and SmallVectorImpl and imports them out of the llvm namespace.